### PR TITLE
Add icon for Zoom plugin

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Screenshot from 2020-04-01 07-26-54

#### Ticket Link
https://github.com/mattermost/mattermost-plugin-zoom/issues/94

### CODE REVIEW

File: cmd/generator/main.go [hunk 1]
```diff
@@ -86,18 +86,18 @@ var generatorCmd = &cobra.Command{
...
 			"mattermost-plugin-aws-SNS",
+			"mattermost-plugin-custom-attributes",
+			"mattermost-plugin-github",
 			"mattermost-plugin-gitlab",
+			"mattermost-plugin-jira",
+			"mattermost-plugin-jenkins",
 			"mattermost-plugin-nps",
 			"mattermost-plugin-webex",
+			"mattermost-plugin-welcomebot",
+			"mattermost-plugin-zoom",
```
Comment (hanzei) [inline cmd/generator/main.go:100]: I did took the opportunity to order the repositories alphabetically.

**Comment by hanzei** [comment]
Head up that this PR goes directly into production.

File: cmd/generator/main.go [hunk 2]
```diff
@@ -107,6 +107,7 @@ var generatorCmd = &cobra.Command{
 			"mattermost-plugin-jenkins": "data/icons/jenkins.svg",
 			"mattermost-plugin-jira":    "data/icons/jira.svg",
 			"mattermost-plugin-webex":   "data/icons/webex.svg",
+			"mattermost-plugin-zoom":    "data/icons/zoom.svg",
```
Comment (lieut-data) [inline cmd/generator/main.go:110]: Hosting the images locally was just meant as a stop-gap until the plugins defined the icons themselves, and posted a release containing same. Could we just wait for that, and slowly reduce this list to zero instead?

Comment (hanzei) [reply cmd/generator/main.go:110]: Given that there are no "old" release of the zoom plugin that need the icon, I agree that it's not worth continuing to use this workaround. I will close the PR.

**Pull request closed without merging**